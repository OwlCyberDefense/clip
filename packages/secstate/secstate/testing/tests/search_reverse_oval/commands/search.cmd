/usr/sbin/secstate search -r oval:com.tresys.oval.rhel:def:1000
