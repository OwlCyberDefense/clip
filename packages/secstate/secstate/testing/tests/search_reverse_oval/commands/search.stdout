In benchmark password-req:
OVAL Definition oval:com.tresys.oval.rhel:def:1000 is used by password-change-time
In benchmark PassComp:
OVAL Definition oval:com.tresys.oval.rhel:def:1000 is used by PassComp-R-2-1
